modules/manage/partials/iceberg/about-iceberg-topics.adoc

```diff
@@ -0,0 +1,476 @@
+
+// image::shared:iceberg-integration-optimized.png[Redpanda's Iceberg integration]
+image::ROOT:shared:iceberg-integration-optimized.png[Redpanda's Iceberg integration]
+
+When you enable the Iceberg integration for a Redpanda topic, Redpanda brokers store streaming data in the Iceberg-compatible format in Parquet files in object storage, in addition to the log segments uploaded using Tiered Storage. Storing the streaming data in Iceberg tables in the cloud allows you to derive real-time insights through many compatible data lakehouse, data engineering, and business intelligence https://iceberg.apache.org/vendors/[tools^]. 
+
+== Prerequisites
+
+To enable Iceberg for Redpanda topics, you must have the following:
+
+ifdef::env-byoc[]
+* A running xref:get-started:cluster-types/byoc/index.adoc[BYOC] cluster. The Iceberg integration is supported only for BYOC.
+* *rpk*: See xref:get-started:rpk-install.adoc[].
+* Familiarity with the Redpanda Cloud API. You must xref:redpanda-cloud:manage:api/cloud-api-authentication.adoc[authenticate] to the Cloud API and use the Control Plane API to update your cluster configuration.
+endif::[]
+
+ifndef::env-byoc[]
+* *rpk*: See xref:get-started:rpk-install.adoc[].
+* *Enterprise license*: To check if you already have a license key applied to your cluster:
++
+[,bash]
+----
+rpk cluster license info
+----
+* *Tiered Storage*: Enable xref:manage:tiered-storage.adoc#set-up-tiered-storage[Tiered Storage] for the topics for which you want to generate Iceberg tables.
+endif::[]
+
+== Limitations
+
+* It is not possible to append topic data to an existing Iceberg table that is not created by Redpanda.
+* If you enable the Iceberg integration on an existing Redpanda topic, Redpanda does not backfill the generated Iceberg table with topic data.
+* JSON schemas are not currently supported. If the topic data is in JSON, use the `key_value` mode to store the JSON in Iceberg, which then can be parsed by most query engines.
+* If you're using Avro or Protobuf data, you must use the Schema Registry wire format, where producers include the magic byte and schema ID in the message payload header. See also: 
+ifndef::env-byoc[pass:m[xref:manage:schema-reg/schema-id-validation.adoc] and] 
+the https://www.redpanda.com/blog/schema-registry-kafka-streaming#how-does-serialization-work-with-schema-registry-in-kafka[Understanding Apache Kafka Schema Registry^] blog post to learn more about the wire format.
```

Comment:
Please fix xref & check whether this blog really explains the wire format

Reply:
Updated